Assignment-2/src/main/java/models/UserClass.java

```diff
@@ -0,0 +1,73 @@
+package models;
+
+import java.util.Arrays;
+
+/**
+ * User class.
+ */
+public class UserClass {
+  /**
+   * name field.
+   */
+  private String name;
+
+  /**
+   * age field.
+   */
+  private int age;
+
+  /**
+   * address field.
+   */
+  private String address;
+
+  /**
+   * roll number field.
+   */
+  private int roll;
+
+  /**
+   * courses field.
+   */
+  private final char[] courses = new char[4];
+
+  public int getRoll() {
+    return roll;
+  }
+
+  public void setRoll(int roll) {
+    this.roll = roll;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public String getAddress() {
+    return address;
+  }
+
+  public void setAddress(String address) {
+    this.address = address;
+  }
+
+  public char[] getCourses() {
+    return Arrays.copyOf(courses, courses.length);
+  }
+
+  public void setCourses(char... courses) {
+    System.arraycopy(courses, 0, this.courses, 0, courses.length);
+  }
```

Comment:
you don't have to manually write getters and setters.
IDEs usually have a feature to generate these.

Reply:
they generated that but here it is using arrays and what i did was suggested by the checkstyle